### ISSUE TITLE
if a pub can't download a file when it first hears about it, it never does

### ISSUE THREAD

**Issue by mmckegg** [mmckegg/ferment#47]
A pub should keep trying to download files from whitelisted people

**Comment by ahdinosaur** [comment]
@mmckegg ohhh, is this what happened with `@nilon`'s files?

**Comment by mmckegg** [comment]
Yeah I think so. I am now able to play them because I happened to be online at the same time as `@nilon` once, but the pubs will never be interested in pulling them. 

Most of the problem is still because we can't seed more than a few torrents at a time without the [cpu usage exploding](https://github.com/feross/webtorrent/issues/903). Ideally, we'd just seed everything as much as possible, all the time.
